inst/pages/12_quality_control.qmd

```diff
@@ -25,8 +25,8 @@ Abundance visualization is an important data exploration
 approach. `miaViz` integrated the `plotAbundanceDensity` function to plot
 the most abundant taxa along with several options.
 
-Next, a few demonstrations are shown, using the [@Lahti2014]
-dataset. A Jitter plot based on relative abundance data, similar to
+Next, a few demonstrations are shown using the [@Lahti2014]
+dataset. A jitter plot based on relative abundance data, similar to
 the one presented at [@Salosensaari2021] supplementary figure 1, can
```

Comment:
put supplementary figure 1 in parenthese and write (Supplementary Fig. 1)

Reply:
Sure, done :)) Can it be merged now?